CMakeLists.txt

```diff
@@ -135,7 +135,10 @@ foreach(var ${variables})
 endforeach()
 
 #---Make sure the Jupyter ROOT C++ kernel runs with the same Python version as ROOT-----
-configure_file(${CMAKE_SOURCE_DIR}/etc/notebook/kernels/root/kernel.json.in ${CMAKE_BINARY_DIR}/etc/notebook/kernels/root/kernel.json)
+set(root_kernel_dir  etc/notebook/kernels/root)
+set(root_kernel_file kernel.json)
+configure_file(${CMAKE_SOURCE_DIR}/${root_kernel_dir}/${root_kernel_file}.in ${CMAKE_BINARY_DIR}/${root_kernel_dir}/${root_kernel_file})
+install(FILES ${CMAKE_BINARY_DIR}/${root_kernel_dir}/${root_kernel_file} DESTINATION ${CMAKE_INSTALL_PREFIX}/${root_kernel_dir})
```

Comment:
Please do not hard-code `etc/...`, but use `${CMAKE_INSTALL_SYSCONFDIR}` whenever possible. Otherwise when someone customizes the install path it won't work. Even better would be to place this sort of file under `<prefix>/share/root` by using `${CMAKE_INSTALL_DATADIR}`, though.

Reply:
Done, thanks for the tip!

Reply:
Actually, I think you can keep it as `etc/...` and omit the `install(FILES ...)` command. The directory `${CMAKE_BINARY_DIR}/etc` is already installed [here](https://github.com/root-project/root/pull/2952/files/ddef8d94ab556ab40e086307342427bef3c38d4e..96001386eb6ce7f6a7fe427cc3ff96add74c6552#diff-af3b638bc2a3e6c650974192a53c7291R420) into the right place. My suggested change is
```
configure_file(etc/notebook/kernels/root/kernel.json.in
               etc/notebook/kernels/root/kernel.json)
```
as `${CMAKE_SOURCE_DIR}` and `${CMAKE_BINARY_DIR}` should be already implied.

Reply:
Noted, will simplify it like you suggest.